src/swap/components/Swap.tsx

```diff
@@ -213,6 +213,11 @@ export function Swap({ address, children, title = 'Swap' }: SwapReact) {
     roundedFromTokenBalance,
     roundedToTokenBalance,
     swapQuoteLoadingState,
+    setFromAmount,
+    setFromToken,
+    setSwapQuoteLoadingState,
+    setToAmount,
+    setToToken,
```

Comment:
if we don't add these the `fromAmount` and `toAmount` changes aren't reflected in `SwapButton`

Reply:
but it is causing lint to fail

Reply:
For those edge cases, just add 
```
/* biome-ignore lint: [add reason] */
```

Reply:
5% of the time Linting rules are not efficient, and for those cases we ignore them and document the reason